src/components/Main/Main.js

```diff
@@ -41,6 +47,13 @@ export class MainComponent {
     }
   }
 
+  handleArtistsView(e) {
```

Comment:
please use event delegation to handle click (it is already implemented in handleListClick)

Reply:
fixed